pkgs/development/python-modules/python-lorem/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  python3Packages,
+}:
+buildPythonPackage rec {
+  pname = "python-lorem";
+  version = "1.3.0.post3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "python_lorem";
```

Comment:
```suggestion
```

Not sure why you would do this, https://pypi.org/project/python_lorem/ is redirected to https://pypi.org/project/python-lorem/

Reply:
oh okay, weird